### DISCUSSION TITLE
Approved for the GitHub Student Developer Pack, but benefits remain inaccessible as your application is still being processed. Once completed, you will have access to the offers.

### DISCUSSION THREAD

**Discussion by rehanaslamtl** [community/community#142523] [GitHub Education] [answered]
### Hi everyone,

Hi everyone,

I’m reaching out because I was recently accepted into the GitHub Student Developer Pack

, but I still don’t have access to its benefits. I know it can take some time for everything to get processed—sometimes up to five days or more—but I wanted to see if anyone could help expedite this.

I’ve double-checked my account, and unfortunately, I can’t access any of the offers that come with the pack. I’m really looking forward to utilizing these benefits for my projects and coursework.

Is there anything specific I should do, or is this delay typical? Any insights or support would be greatly appreciated!

Thanks so much!

![{F0842C1A-ED08-4ED0-B108-A6D3BD77367B}](https://github.com/user-attachments/assets/471b9113-9aae-4592-b415-3ad7c975f717)

**Comment by davevad93** [comment]
Hi @rehanaslamtl ,

Don't worry, it's normal and you don't have to do anything., the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @rehanaslamtl was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by LucasTKP** [comment]
@rehanaslamtl Já liberaram seus benefícios? estou na mesma situação

**Comment by rehanaslamtl** [comment]
Hey not yet but i believe within a week as far i have looked into other people's ticketsGet Outlook for AndroidFrom: Lucas Gean dos Santos ***@***.***>Sent: Friday, October 25, 2024 9:46:15 pmTo: community/community ***@***.***>Cc: Muhammad Rehan Aslam ***@***.***>; Mention ***@***.***>Subject: Re: [community/community] Approved for the GitHub Student Developer Pack, but benefits remain inaccessible as your application is still being processed. Once completed, you will have access to the offers. (Discussion #142523)
@rehanaslamtl Já liberaram seus benefícios? estou na mesma situação

**Comment by rehanaslamtl** [comment, marked as answer]
hey i have got my approved just after waiting for few days thanks everyone!
![{25980DEC-026F-4753-91F2-02F45662594B}](https://github.com/user-attachments/assets/034067ee-a344-422f-92e2-14286af6ac79)
